pystan/misc.py

```diff
@@ -941,6 +946,7 @@ def _writable_sample_file(file, warn=True, wfun=None):
     if wfun is None:
         wfun = lambda x, y: '"{}" is not writable; use "{}" instead'.format(x, y)
     dir = os.path.dirname(file)
+    dir = os.getcwd() if dir == '' else dir
```

Comment:
When `file` is just the name of a file, `os.path.dirname(file)` returns `''`, which was failing on the next call `os.access(dir, os.W_OK)`. By checking if it's `''` and returning the current working directory, this now allows the files to get written to the working directory. Is that ok?